packages/react-native-web/src/exports/View/ViewPropTypes.js

```diff
@@ -74,7 +74,8 @@ export type ViewProps = {
   onMagicTap?: Function,
   removeClippedSubviews?: boolean,
   renderToHardwareTextureAndroid?: boolean,
-  shouldRasterizeIOS?: boolean
+  shouldRasterizeIOS?: boolean,
+  tvParallaxProperties?: any
```

Comment:
This type is different to the one on L127

Reply:
Sorry - I've never used flow before and thought that was the appropriate type in this case as we'd never actually use this prop in RNW so weren't too concerned what was in it? If you let me know what it should be instead I'd be happy to fix it and reopen. Thanks!